src/tests/cxxfrontend.c

```diff
@@ -525,12 +527,12 @@ void test_cppmangle()
     fd = (*cd->members)[0]->isFuncDeclaration();
     assert(fd);
     mangle = cppThunkMangleItanium(fd, b->offset);
-    assert(strcmp(mangle, "_ZThn4_N7Derived9MethodCPPEv") == 0);
+    assert(strcmp(mangle, "_ZThn8_N7Derived9MethodCPPEv") == 0);
```

Comment:
The previous thunk offset was 4, because `target.isLP64` wasn't initialized and remained at its initial `false` value [but `target.is64bit` was `true` if the test was run on a 64-bit machine].